content/docs/deployment/csmoperator/_index.md

```diff
@@ -152,6 +153,7 @@ To perform an offline installation, the following steps should be performed:
 >NOTE: It is recommended to use the same build tool for packing and unpacking of images (either docker or podman).
 
 #### Building an offline bundle
+>NOTE: Login to `registry.redhat.io` registry using Redhat credentials before you proceed with offline bundle creation.
```

Comment:
RedHat, uppercase H. Also "Login to **the**..."

Reply:
Fixed